client/src/types/data.ts

```diff
@@ -0,0 +1,109 @@
+import { Number, Record, Static } from "runtypes";
+
+/* ------------------------------------ RunTypes ------------------------------------ */
+
+/** Value runtype of temperature sensor data */
+export const TemperatureRT = Number;
+
+/** Value runtype of humidity sensor data */
+export const HumidityRT = Number;
+
+/** Value runtype of steeringAngle sensor data */
+export const SteeringAngleRT = Number;
+
+/** Value runtype of co2 sensor data */
+export const CO2RT = Number;
+
+/** Value runtype of accelerometer sensor data */
+export const AccelerometerRT = Record({
+    /** X value */
+    x: Number,
+    /** Y value */
+    y: Number,
+    /** Z value */
+    z: Number
+});
+
+/** Value runtype of gyroscope sensor data */
+export const GyroscopeRT = Record({
+    /** X value */
+    x: Number,
+    /** Y value */
+    y: Number,
+    /** Z value */
+    z: Number
+});
+
+/** Value runtype of reedVelocity sensor data */
+export const ReedVelocityRT = Number;
+
+/** Value runtype of reedDistance sensor data */
+export const ReedDistanceRT = Number;
+
+/** Value runtype of gps sensor data */
+export const GPSRT = Record({
+    /** Speed */
+    speed: Number,
+    // /** Dilution of precision */
+    // pdop: Number,
+    /** Latitude */
+    latitude: Number,
+    /** Longitude */
+    longitude: Number,
+    /** Altitude */
+    altitude: Number,
+    /** Course */
+    course: Number,
+    // /** Datetime */
+    // datetime: String
```

Comment:
The notion page gives specs for the gps sensor data including pdop and datetime. Is that something we're adding in later?

Reply:
a775dc8

I didn't specify a V3 version as for V2 all the sensor data is clumped together anyway.